e2e/plugin-api/package.json

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "plugin-api-test",
+  "type": "module"
```

Comment:
I want this e2e tests to have several things:
- one folder for `ts`
- one folder for `cjs`
- one folder for `mjs`

You can reference https://github.com/ahnpnl/jest/tree/main/e2e/esm-config for examples